suzieq/poller/controller/controller.py

```diff
@@ -108,6 +108,9 @@ def __init__(self, args: argparse.Namespace, config_data: dict) -> None:
                         'no-coalescer': self._no_coalescer,
                         'output-dir': args.output_dir,
                         'outputs': args.outputs,
+                        # We are intentionally passing the run_once argument
+                        # since we do not want to give internal values as
+                        # arguments of the poller workers
```

Comment:
Claudio, why do you want to use run-once argument. Its not even running once in this case. Why not use --debug?